tests/layers/test_nms_rotated.py

```diff
@@ -87,6 +88,13 @@ def test_batched_nms_rotated_0_degree_cpu(self):
             backup = boxes.clone()
```

Comment:
Can you create a separate test method to test scriptability? Also this class and this file can be renamed to just NMS, since it's no longer specific to NMSRotated.

Reply:
Got it. I will create `test_nms.py` and move this ut to it.